libraries/AP_ESC_Telem/AP_ESC_Telem_SITL.cpp

```diff
@@ -46,27 +58,13 @@ void AP_ESC_Telem_SITL::update()
             .consumption_mah = 1.0f,
         };
 
-        update_telem_data(i, t,
+        update_telem_data(motor, t,
             AP_ESC_Telem_Backend::TelemetryType::CURRENT
                 | AP_ESC_Telem_Backend::TelemetryType::VOLTAGE
                 | AP_ESC_Telem_Backend::TelemetryType::CONSUMPTION
                 | AP_ESC_Telem_Backend::TelemetryType::TEMPERATURE);
     }
-
-    if (is_zero(sitl->throttle)) {
-        if (!is_zero(sitl->esc_rpm_armed) && hal.util->get_soft_armed()) {
-            for (uint8_t i = 0; i < sitl->state.num_motors; i++) {
-                update_rpm(i, sitl->esc_rpm_armed);
```

Comment:
This is a test parameter used by the takeoff checks - need it in here somehwere

Reply:
fixed, thanks